src/powershell-native/nativemsh/pwrshplugin/pwrshclrhost.cpp

```diff
@@ -270,7 +270,10 @@ unsigned int PowerShellClrWorker::LoadWorkerCallbackPtrs(
     {
         PWSTR msg = NULL;
         exitCode = g_MANAGED_METHOD_RESOLUTION_FAILED;
-        GetFormattedErrorMessage(&msg, exitCode);
+        /* NOTE: dont' use a string literal for a fallback; PlugInException expects msg to allocated
```

Comment:
This comment seems unnecessary. The behavior is documented in the class definition.

Reply:
I added this comment because the constructor annotations were wrong and, since the return value of GetFormmatedErrorMessage was being ignored, I added the comment to document the behavior as intentional.